src/container/containerManager.ts

```diff
@@ -113,27 +113,27 @@ export class ContainerManager {
 
     private async generateDeploymentString(templateFile: string,
```

Comment:
`generateDeploymentString` now does not generate a string anymore. Suggest updating the name to reflect this.